ygot/render.go

```diff
@@ -839,7 +840,14 @@ func constructJSON(s GoStruct, parentMod string, args jsonOutputConfig) (map[str
 			continue
 		}
 
-		value, err := constructJSONValue(field, pmod, args)
+		var value interface{}
+
+		if util.IsYgotAnnotation(fType) {
+			value, err = constructJSONAnnotationSlice(field)
```

Comment:
(optional) The word "construct" feels redundant in these names. Here and below.

Reply:
True -- refactored all the `construct*` names other than the public API functions.